packages/sdk-js/src/handlers/auth.ts

```diff
@@ -18,13 +18,19 @@ export class AuthHandler {
 	private storage: AuthStorage;
 	private mode: 'cookie' | 'json';
 	private autoRefresh: boolean;
+	private autoRefreshTimeout?: any;
+	private expiresAt?: number;
+	/** Used for tracking if accessToken is restored from store to config */
+	private accessTokenInitialized: boolean = false;
 
 	constructor(axios: AxiosInstance, options: AuthOptions) {
 		this.axios = axios;
 		this.storage = options.storage;
 		this.mode = options.mode;
 		this.autoRefresh = options.autoRefresh;
 
+		this.axios.interceptors.request.use((config) => this.initializeAccessToken(config));
```

Comment:
Instead of doing this every request, would it be possible to do this only when the SDK is initialized the first time?

Reply:
Added `this.axios.interceptors.request.eject` in `initializeAccessToken`